test/acceptance/member_test.exs

```diff
@@ -0,0 +1,43 @@
+defmodule Community.Acceptance.MemberTest do
+  use Community.AcceptanceCase
+  alias Community.Member
+
+  hound_session
+
+  # test "homepage works" do
```

Comment:
I agree that would be better but unfortunately, it won't automatically skip them so if someone simply runs `mix test`, it will jack up their entire DB. For now I'm going to stick with comments.